### DISCUSSION TITLE
Can't login!

### DISCUSSION THREAD

**Discussion by claudedubouzet** [codemagic-ci-cd/codemagic-docs#2040] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Stuck at Authenticating after logging in using bitbucket or gitlab. What is wrong with this? Already cleared cache and  tried logging in using different devices.

### Expected results

Login

### Actual results

Can't Login

**Comment by himesh-cm** [comment]
Hi @FriendlyNeighbor, Can you please help with the team id?

**Reply by claudedubouzet** [reply]
I'm confused why does it matter. I have 2 different accounts and projects. 1 project in bitbucket and another in gitlab and they are also opened in a separate browse. Why all of a sudden both accounts are stuck in "Authenticating..."

I've tried using different browsers, even on my mobile's browser, flushed my dns and reset my ip. This is so frustrating.

**Reply by himesh-cm** [reply]
I do not have your account details to check and investigate the cause. If you could share the details, i will look into it and get back to you.

**Reply by claudedubouzet** [reply]
goleansigma

**Reply by claudedubouzet** [reply]
I even tried Email Login out of curiosity and the login button doesn't do anything

**Reply by claudedubouzet** [reply]
It just shows "The connection seems slow" I have 250Mbps internet

**Reply by claudedubouzet** [reply]
I'm gonna list here the other workspace id that I've connected codemagic into
Bitbucket: 
Bitbucket: the-church-online
Gitlab: (I'm not familiar with the team ID you mentioned but here's the link) 
https://gitlab.linkpoint.ca/flowerroad/mobile-app 
and the project id is 55

**Reply by himesh-cm** [reply]
Can you please confirm if you are facing the same issue in incognito mode?

**Reply by claudedubouzet** [reply]
yes

**Reply by dtrdic** [reply]
Can you please check on https://github.com/settings/installations and see Codemagic CI/CD app access is not suspended or some other error is displayed?
Look also if Codemagic CI/CD is under the Authorized tab. Also please check your emails, spam or junk folder.
<img width="977" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/f2d5fb9e-edd5-4d70-8d0a-2f80af9e0f5f">

**Reply by claudedubouzet** [reply]
I don't use github. I use bitbucket and gitlab

**Comment by ejbdobp** [comment]
me too, I cant login to codemagic website using email.

**Reply by dtrdic** [reply]
hey @ejbdobp are you still facing login issues?

**Reply by ejbdobp** [reply]
not anymore, thank you. it seems the api.codemagic.com is back up

**Comment by claudedubouzet** [comment]
I just spent additional monthly cost and used a vpn. but I hope this can be resolved. this could be an isp issue

**Reply by himesh-cm** [reply]
Hi @FriendlyNeighbor, Do you mean you can log in using a VPN?

**Comment by dtrdic** [comment]
Here's for [gitlab](https://gitlab.com/-/profile/applications)
try revoke and authorize again
